metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -428,16 +428,13 @@ def get_properties_workunit(
         sObjectProperties = {
             propertyLabels[k]: str(v)
             for k, v in sObject.items()
-            if k in propertyLabels.keys() and v is not None
+            if k in propertyLabels and v is not None
+        } | {
```

Comment:
Reverted this usage of merge(|) operator as this is not supported below python 3.9.